Cubical/Categories/Monoidal/Base.agda

```diff
@@ -0,0 +1,121 @@
+-- Monoidal categories
+{-# OPTIONS --safe #-}
+
+module Cubical.Categories.Monoidal.Base where
+
+open import Cubical.Categories.Category.Base
+open import Cubical.Categories.Constructions.Product
+open import Cubical.Categories.Functor.Base
+open import Cubical.Categories.Functor.Product
+open import Cubical.Categories.Morphism
+open import Cubical.Categories.NaturalTransformation.Base
+open import Cubical.Foundations.Prelude
+
+
+module _ {ℓ ℓ' : Level} (C : Category ℓ ℓ') where
+  open Category C
+
+  private
+    record TensorStr : Type (ℓ-max ℓ ℓ') where
+      field
+        ─⊗─ : Functor (C × C) C
+        i : ob
+
+      open Functor
+
+      -- Useful tensor product notation
+      _⊗_ : ob → ob → ob
+      x ⊗ y = ─⊗─ .F-ob (x , y)
+
+      _⊗ₕ_ : ∀ {x y z w} → Hom[ x , y ] → Hom[ z , w ]
+           → Hom[ x ⊗ z , y ⊗ w ]
+      f ⊗ₕ g = ─⊗─ .F-hom (f , g)
+
+
+  record StrictMonStr : Type (ℓ-max ℓ ℓ') where
+    field
+      tenstr : TensorStr
+
+    open TensorStr tenstr public
+
+    field
+      -- Axioms - strict
+      assoc : ∀ x y z →  x ⊗ (y ⊗ z) ≡ (x ⊗ y) ⊗ z
+      idl : ∀ x →  i ⊗ x ≡ x
+      idr : ∀ x →  x ⊗ i ≡ x
+
+
+  record MonoidalStr : Type (ℓ-max ℓ ℓ') where
+    field
+      tenstr : TensorStr
+
+    open TensorStr tenstr public
+
+    private
+      -- Give some names to things
```

Comment:
These names are not great... Do they get exported somehow or are they just local?

Reply:
Nope, I checked and they are not exported. They are just there to make the "axioms" below look nice (like equations).